docs/_layouts/default.html

```diff
@@ -23,7 +23,7 @@ <h1 id="site-title">{{ site.title | default: site.github.repository_name }} <a c
       <p>{{ site.description | default: site.github.project_tagline }}</p>
     </div>
 
-    <p><small>&copy; 2018 <a href="{{ site.github.owner_url }}">Domizio Demichelis</a> &mdash; <a href="https://opensource.org/licenses/MIT">MIT License</a></small></p>
+    <p><small>&copy; 2019 <a href="{{ site.github.owner_url }}">Domizio Demichelis</a> &mdash; <a href="https://opensource.org/licenses/MIT">MIT License</a></small></p>
```

Comment:
Thank you!!! I am always slow to pick the new year! :)
Actually it should be 2017-2019

Reply:
Fixed :)